features/mesh/Index.feature

```diff
@@ -32,8 +37,14 @@ Feature: mesh / index
           gateway:
             total: 10
       """
-    When I visit the "/" URL
-    When I click the "$mesh-selector" element and select "aalphabetically-second-because-of-default"
-    Then the "$mesh-breadcrumb" element contains "aalphabetically-second-because-of-default"
-    When I click the "$mesh-selector" element and select "default"
-    Then the "$mesh-breadcrumb" element contains "default"
+
+    When I visit the "/meshes" URL
+
+    When I click the "<Selector>" element
+    Then the URL contains "/mesh/<Mesh>"
+    And the "$mesh-breadcrumb" element contains "<Mesh>"
+
+    Examples:
+      | Mesh                                      | Selector                                         |
+      | aalphabetically-second-because-of-default | [data-testid='data-overview-table'] tbody tr:nth-child(2) [data-testid='detail-view-link'] |
+      | default                                   | [data-testid='data-overview-table'] tbody tr:nth-child(1) [data-testid='detail-view-link'] |
```

Comment:
Should we make CSS selector aliases for these? We can probably get rid of the ridiculously named mesh now 😆 .

Also did I see this test no longer requires `/mesh-insights` mocking out? Or does it still need it but only need at least 1 item maybe?  not sure lmk.

Reply:
I renamed the second mesh and removed the mesh insights mock. Adding aliases here doesn’t really help, does it?

Reply:
> Adding aliases here doesn’t really help, does it?

Well if we are in here it reduces the duplication, up to you, no biggie.

Reply:
But how? We could have a partial alias for the start and end bit of the overall selector, but it’s the middle bit that’s different.

Reply:
Something like:

https://github.com/kumahq/kuma-gui/blob/8cdae53804e11491ff218fb81551f3401931a5c3/features/mesh/dataplanes/Index.feature#L7-L9

https://github.com/kumahq/kuma-gui/blob/8cdae53804e11491ff218fb81551f3401931a5c3/features/mesh/dataplanes/Index.feature#L69-L72